protocol/x/clob/keeper/process_single_match.go

```diff
@@ -243,6 +246,25 @@ func (k Keeper) ProcessSingleMatch(
 			notionalLiquidatedQuoteQuantums,
 			takerInsuranceFundDelta,
 		)
+
+		labels := []gometrics.Label{
+			metrics.GetLabelForIntValue(metrics.PerpetualId, int(perpetualId)),
+			metrics.GetLabelForBoolValue(metrics.IsBuy, matchWithOrders.TakerOrder.IsBuy()),
+			metrics.GetLabelForBoolValue(metrics.CheckTx, ctx.IsCheckTx()),
+		}
+
+		// Stat quote quantums liquidated.
+		telemetry.IncrCounterWithLabels(
+			[]string{metrics.Liquidations, metrics.PlacePerpetualLiquidation, metrics.Filled, metrics.QuoteQuantums},
```

Comment:
Should you include CLOB module name as a prefix in this stat?

Reply:
Lol I was actually going to drop `metrics.Liquidations`. 

I think having too many strings here can have perf implications because of string concatenation. so let's minimize the number of strings if possible. let me know what you think.

Reply:
Ideally we have the module name as a prefix. Maybe we can add the CLOB prefix to `metrics.PlacePerpetualLiquidation`? Up to you though.